core/test/audits/mainthread-work-breakdown-test.js

```diff
@@ -101,6 +104,26 @@ describe('Performance: page execution timings audit', () => {
     assert.equal(output.score, 1);
   });
 
+  it('should compute the correct values for the load trace', async () => {
```

Comment:
really any modern chrome trace will do here, which is why I didn't bother making a user flow for a basic page. instead just used `animations` here.